agent/arbiter-agent.c

```diff
@@ -267,6 +267,20 @@ chamge_arbiter_agent_handle_deactivate (ChamgeDBusArbiterManager * manager,
   return TRUE;
 }
 
+static gboolean
+_is_enrolled (GList * list, const char *id)
+{
+  GList *edge;
+  for (edge = list; edge != NULL; edge = g_list_next (edge)) {
```

Comment:
You might want to use `g_list_find_custom ()`

Reply:
I will keep in mind and fix it later.